src/mixpanel.js

```diff
@@ -23,6 +23,7 @@ const _ = require('lodash')
 
 module.exports = (MixpanelLib) => {
   const env = detect.getName()
+  let projectToken = null
```

Comment:
Not that it's necessary, but why not stored this on `properties`?

Reply:
Good call! Fixed